### DISCUSSION TITLE
Not liking the feed, want default repos again

### DISCUSSION THREAD

**Discussion by TonyGravagno** [community/community#51029] [Other Feature Feedback, Questions, & Ideas]
Product Feedback

The feed doesn't have anything of interest to me. I wouldn't mind going to the various filters when I have time. But when I come here I want to see my repositories, not what someone else has done. Rather, I'm sure some people Want to see things that are in the feed. But it's now an extra step to get to what I want, which used to be the default.

This was a bad UX decision - completely changing something without notice. A better decision would have been to make it optional, and then making it dirt-simple to get to our repositories.

To be clear, sure, we see all repos in the left sidebar - I just want to see mine .... until I decide to see others.

TY

**Comment by adlai** [comment]
I suggest bookmarking your desired landing page, instead of whatever the webserver barfs when you GET /
